src/core/unif.ml

```diff
@@ -33,6 +33,30 @@ let pp_problem oc p =
 let no_problems : problems =
   {to_solve  = []; unsolved = []; recompute = false}
 
+
+(** Definition of symbols used to encode unification rules. *)
+module Hint = struct
+
+  let sign =
+    let pth = [""; "unif_rule"] in
+    let s = Sign.create pth in
+    s.sign_deps := Files.PathMap.empty;
+    Sign.loaded := Files.PathMap.add pth s !(Sign.loaded);
+    s
+
+  (** Symbol representing an atomic unification problem. The term [atom t u]
+      represents [t ≡ u]. The left-hand side of a unification rule is made
+      of only one atom. *)
+  let atom : sym =
+    Sign.add_symbol sign Public Defin (Pos.none "u_atom") Kind []
```

Comment:
Why do you want to add a prefix "u_"?

Reply:
Otherwise we miss some information when symbols are printed out. Except if we print them fully qualified

Reply:
I mean, getting for instance `atom symbol not found` may be confusing

Reply:
In this case, I would prefer a prefix that is not allowed by the parser, say "#" for instance. Here, "#equiv" and "#list". What do you think?

Reply:
Good idea.